modules/ROOT/pages/managing-connected-salesforce-orgs.adoc

```diff
@@ -0,0 +1,96 @@
+= Managing Connected Salesforce Orgs
+
+As an Anypoint Platform organization administrator, you can enable, disable, remove, and repair connections, and edit the name of Salesforce organizations.
+
+[enable-disable-connection]
+== Enable or Disable a Connection in Anypoint Platform
+
+Enable or disable the connection to a Salesforce organization that has an established tenant relationship with your Anypoint Platform organization. 
+
+[NOTE]
+Tenant relationships that are automatically provisioned for your Anypoint Platform organization are disabled until you enable them. 
+
+Disabling the connection between Anypoint Platform and a Salesforce organization doesn't remove the connection to the Salesforce organization. 
+The Salesforce organization still appears in the list of available organizations with a status of `Disabled`. 
+You can't publish assets from Anypoint Platform to the Salesforce organization or use any of the other features associated with the connected Salesforce organization until the connection is reenabled.  
+
+include::partial$include-nav-salesforce.adoc[]
+
++
+If your organization has at least one established tenant relationship with a Salesforce organization, a list of Salesforce orgs appears, showing the name, organization ID, and status of the tenant relationship connection.
+. Switch between *Enable* and *Disable* for the Salesforce organization to enable or disable the connection for.
+
+[[removing-connections]]
+== Remove the Connection Between Anypoint Platform and Salesforce Organizations
+
+When you remove the connection between your Anypoint Platform organization and Salesforce organization, the Salesforce organization no longer appears in the list of connected organizations. The connection still exists in the Salesforce organization until it's removed by the Salesforce organization administrator.
+
+To reestablish the connection between Anypoint Platform and removed Salesforce organizations, administrators with the appropriate permissions in both Anypoint Platform and Salesforce organizations must recreate the connection.
+
+[removing-connection-anypoint-platform]
+=== Remove a Connected Salesforce Org in Anypoint Platform
+
+An Anypoint Platform organization administrator can remove the connection between their Anypoint Platform organization and a connected Salesforce organization.
+
+When you remove a connection to a Salesforce organization, any capabilities that are associated with the Salesforce organization, such as Einstein, are removed. 
+
+To remove the connection to a Salesforce organization:
+
+include::partial$include-nav-salesforce.adoc[]
+
+. Click the *Actions* menu (*...*) in the Salesforce organization to remove the connection from and select *Remove Salesforce Org*. 
```

Comment:
I still think you don't need "in the Salesforce org to remove the connection from and" because that's part of the context.

Reply:
I don't know. There can be multiple Salesforce orgs listed and each one has its own action menu.